spec/lib/plug/configuration_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe Plug::Configuration do
+  let(:buttons_config) { [{ label: 'Clear Cache', task: 'tmp:cache:clear' }] }
+
+  it 'allows the addition of buttons to the configuration' do
```

Comment:
No need to mention `to the configuration` because the test is describing the `Configuration` class.